src/app/view-modes/collation/collation.component.html

```diff
@@ -1,11 +1,7 @@
 <gridster [options]="options">
     <gridster-item [item]="textPanelItem">
-        <evt-text-panel
-            [pageID]="currentPageID$ | async" 
-            (pageChange)="changePage($event)"       
-            [hideEditionLevelSelector]="true" 
-            [editionLevelID]="'critical'"
-        ></evt-text-panel>
+        <evt-critical-text-panel
+            [editionLevelID]="'critical'"></evt-critical-text-panel>
```

Comment:
Since you use a specific component that is `evt-critical-text-panel` you probably don't need to pass the editionLevelID as input. For this specific panel I expect the edition level will never change and always be "critical". Can't you define it directly in the component?